core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -420,6 +420,21 @@ package object config {
       .booleanConf
       .createWithDefault(false)
 
+  private[spark] val STORAGE_SHUFFLE_DECOMMISSION_ENABLED =
+    ConfigBuilder("spark.storage.decommission.shuffle_blocks")
```

Comment:
Shall we avoid to use `_` in the config name? For me, `shuffleBlocks` will be more conventional in Apache Spark.

Reply:
Sounds good I'll swap out the _s in the config string for camel

Reply:
Thanks, @Ngone51 and @holdenk .